examples/oci/oci_gpu-sky.yaml

```diff
@@ -1,29 +1,303 @@
+#(gpu-task1, pid=9493) Train Epoch: 1 [46720/60000 (78%)]        Loss: 0.057180
+#(gpu-task1, pid=9493) Train Epoch: 1 [47360/60000 (79%)]        Loss: 0.074373
+#(gpu-task1, pid=9493) Train Epoch: 1 [48000/60000 (80%)]        Loss: 0.046954
+#(gpu-task1, pid=9493) Train Epoch: 1 [48640/60000 (81%)]        Loss: 0.265737
+#(gpu-task1, pid=9493) Train Epoch: 1 [49280/60000 (82%)]        Loss: 0.075507
+#(gpu-task1, pid=9493) Train Epoch: 1 [49920/60000 (83%)]        Loss: 0.083043
+#(gpu-task1, pid=9493) Train Epoch: 1 [50560/60000 (84%)]        Loss: 0.062400
+#(gpu-task1, pid=9493) Train Epoch: 1 [51200/60000 (85%)]        Loss: 0.107635
+#(gpu-task1, pid=9493) Train Epoch: 1 [51840/60000 (86%)]        Loss: 0.128209
+#(gpu-task1, pid=9493) Train Epoch: 1 [52480/60000 (87%)]        Loss: 0.047502
+#(gpu-task1, pid=9493) Train Epoch: 1 [53120/60000 (88%)]        Loss: 0.086729
+#(gpu-task1, pid=9493) Train Epoch: 1 [53760/60000 (90%)]        Loss: 0.068962
+#(gpu-task1, pid=9493) Train Epoch: 1 [54400/60000 (91%)]        Loss: 0.041404
+#(gpu-task1, pid=9493) Train Epoch: 1 [55040/60000 (92%)]        Loss: 0.203918
+#(gpu-task1, pid=9493) Train Epoch: 1 [55680/60000 (93%)]        Loss: 0.144886
+#(gpu-task1, pid=9493) Train Epoch: 1 [56320/60000 (94%)]        Loss: 0.247239
+#(gpu-task1, pid=9493) Train Epoch: 1 [56960/60000 (95%)]        Loss: 0.100990
+#(gpu-task1, pid=9493) Train Epoch: 1 [57600/60000 (96%)]        Loss: 0.118975
+#(gpu-task1, pid=9493) Train Epoch: 1 [58240/60000 (97%)]        Loss: 0.081676
+#(gpu-task1, pid=9493) Train Epoch: 1 [58880/60000 (98%)]        Loss: 0.061787
+#(gpu-task1, pid=9493) Train Epoch: 1 [59520/60000 (99%)]        Loss: 0.059713
+#(gpu-task1, pid=9493)
+#(gpu-task1, pid=9493) Test set: Average loss: 0.0480, Accuracy: 9831/10000 (98%)
+#(gpu-task1, pid=9493)
+#(gpu-task1, pid=9493) The task is completed.
+#INFO: Job finished (status: SUCCEEDED).
+#Shared connection to 129.154.222.76 closed.
+#I 06-10 09:09:33 cloud_vm_ray_backend.py:2847] Job ID: 1
+#I 06-10 09:09:33 cloud_vm_ray_backend.py:2847] To cancel the job:       sky cancel gpu001 1
+#I 06-10 09:09:33 cloud_vm_ray_backend.py:2847] To stream job logs:      sky logs gpu001 1
+#I 06-10 09:09:33 cloud_vm_ray_backend.py:2847] To view the job queue:   sky queue gpu001
+#I 06-10 09:09:33 cloud_vm_ray_backend.py:2962]
+#I 06-10 09:09:33 cloud_vm_ray_backend.py:2962] Cluster name: gpu001
+#I 06-10 09:09:33 cloud_vm_ray_backend.py:2962] To log into the head VM: ssh gpu001
+#I 06-10 09:09:33 cloud_vm_ray_backend.py:2962] To submit a job:         sky exec gpu001 yaml_file
+#I 06-10 09:09:33 cloud_vm_ray_backend.py:2962] To stop the cluster:     sky stop gpu001
+#I 06-10 09:09:33 cloud_vm_ray_backend.py:2962] To teardown the cluster: sky down gpu001
+#Clusters
+#NAME         LAUNCHED    RESOURCES                         STATUS  AUTOSTOP  COMMAND
+#gpu001       4 mins ago  1x OCI(VM.GPU.A10.1, {'A10': 1})  UP      -         sky launch -c gpu001 oci_...
```

Comment:
Let's avoid having the output in the example yaml to keep it simple and easy to read.

Reply:
ok